Automattic/wp-calypso
# Add Akismet API key to thank you page

## Proposed Changes

* This diff adds the Akismet API key to the thank you page following an Akismet purchase

## Testing Instructions

* Navigate to `http://calypso.localhost:3000/checkout/akismet/thank-you/ak_plus_yearly_1` while logged in
* You should see your API key on the thank you page with a button to copy it

## Pre-merge Checklist
